### ISSUE TITLE
the planet tests are requiring a environment key 

### ISSUE THREAD

**Issue by 12rambau** [12rambau/sepal_ui#481]
The key need to test the Planet module may not be provided by the developer (as we cannot force people to have a Planet account) so I would suggest we make them skipable under condition. 

I'm thinking about testing planet key existence, what do you think ?

**Comment by dfguerrerom** [comment]
Yes I do agree with you, although we're making it skippable, it will deserve to be documented in the contribute section.
